### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by gutsytechster**
README.md is updated with more details about project and how one can run the project.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,5 +1,47 @@
...
-There are many drawbacks of the traditional system. Such as, some of the teachers were able to figure out which feedback is given by whom. The evaluation scores had to be fed into the database, which led to wastage of manpower, time and paper. At times it is difficult to understand the writing of the student which makes it troublesome for understanding the suggestion given by the student. 
-So, this project is designed to tackle all these problems efficiently and only admin will be able to see the outcome of the feedback given by students for the respective teachers in a convenient manner.
-But there is also another problem, some students are not regular in colleges but give the feedback due to which it may harm the profile of a good teacher. So, we will also develop our project in such a way that, the feedbacks of such students will not affect the feedback of a teacher.
+## CollegeProject
+
+A project that involves developing a feedback website for college. The project digitalize the feedbacks given by the students for their teachers. There are few drawbacks in traditional system like
+
```
Comment (VirtualRcoder) [inline README.md]: Please remove the extra spaces

File: README.md [hunk 2]
```diff
@@ -1,5 +1,47 @@
...
+    $ python manage.py runserver
+
+Make sure you are in the CollegeProject directory. Go to the localhost and the project would work.
+***
+
+## References
+
+- To know more about Django, refer https://www.djangoproject.com/
+- To know more about Python, refer https://www.python.org/doc/
+
```
Comment (VirtualRcoder) [inline README.md]: All other things are ok

File: README.md (hunk 1 above)
Comment (gutsytechster) [reply README.md]: Hey @VirtualRcoder, we need to keep these spaces in order to distinguish between paragraphs. The spaces here are part of Markdown syntax.

File: README.md (hunk 2 above)
Comment (cypher-naman) [reply README.md]: good to go.

Comment (gutsytechster) [reply README.md]: @cypher-naman, approve it please by clicking on the approve option.

**Pull request merged**